design_drafts/components_architecture_and_urdf_examples.md

```diff
@@ -144,6 +144,287 @@ Note:
   </ros2_control>
 ```
 
+```xml
+  <ros2_control name="12DOF_Legged_System_Robot_MultiInterface" type="system">
+    <hardware>
+      <classType>ros2_control_demo_hardware/2DOF_System_Hardware_MultiInterface_MultiWrite</classType>
+      <param name="example_param_write_for_sec">0.001</param>
+      <param name="example_param_read_for_sec">0.001</param>
+    </hardware>
+    <joint name="front_left_leg_joint1">
+      <classType>ros2_control_components/MultiInterfaceJoint_MultiWrite</classType>
```

Comment:
also capitalized CamelCase:

"MultiInterfaceMultiWriteJoint" (we will rename this in #25)